vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -67,98 +67,79 @@
       </Annotation>
 
 
-      <!-- Metadata annotations -->
-
-      <Term Name="TemporalSupport" Type="Temporal.TemporalSupportType" AppliesTo="EntitySet NavigationProperty">
+      <Term Name="ApplicationTimeSupport" Type="Temporal.ApplicationTimeSupportType" Nullable="false" AppliesTo="Collection">
+        <Annotation Term="Core.AppliesViaContainer" />
         <Annotation Term="Core.Description">
-          <String>This entity set or navigation property supports  temporal requests</String>
+          <String>This entity set or navigation property supports temporal requests</String>
         </Annotation>
       </Term>
 
-      <ComplexType Name="TemporalSupportType">
-        <Property Name="ApplicationTime" Type="Temporal.TimeDimensionType">
-          <Annotation Term="Core.Description" String="Application time granularity" />
+      <ComplexType Name="ApplicationTimeSupportType">
+        <Property Name="CollectionKind" Type="Temporal.CollectionKindType" Nullable="false">
+          <Annotation Term="Core.Description" String="Kind of temporal collection" />
         </Property>
-        <Property Name="SystemTime" Type="Temporal.GranularityDateTimeOffset">
-          <Annotation Term="Core.Description" String="System time granularity" />
+        <Property Name="PeriodKind" Type="Temporal.PeriodKindType" Nullable="false">
+          <Annotation Term="Core.Description" String="Kind of application time period" />
         </Property>
         <Property Name="ObjectKey" Type="Collection(Edm.PropertyPath)" Nullable="false">
           <Annotation Term="Core.Description" String="The set of primitive properties that identify a temporal object" />
-          <Annotation Term="Core.LongDescription" String="A temporal object is a set of facts whose change over application and/or system time is tracked by the service. The entities of the annotated entity set are time slices of temporal objects. A temporal object is uniquely identified within this set by the values of the specified object key properties. Object key properties follow the same rules as entity key properties. If no object key is specified, the object key is identical to the entity key." />
+          <Annotation Term="Core.LongDescription" String="A temporal object is a set of facts whose change over application and/or system time is tracked by the service. The entities of the annotated collection are time slices of potentially multiple temporal objects. A temporal object is uniquely identified within this set by the values of the specified object key properties. Object key properties follow the same rules as entity key properties. If no object key is specified, the annotated collection only contains time slices for a single one temporal object." />

```

Comment:
```suggestion
          <Annotation Term="Core.LongDescription" String="A temporal object is a set of facts whose changes over application time are tracked by the service. The entities in the annotated collection belong to potentially multiple temporal objects, and each temporal object is uniquely identified by the values of the specified object key properties. Object key properties follow the same rules as entity key properties. If no object key is specified, only a single temporal object belongs to the annotated collection." />
```
Avoid the term "time slice" here, because it is defined only for time-slice collections.

Remove "within this set", because this sounds like the set of primitive properties.